cmd/common/config.go

```diff
@@ -217,28 +245,16 @@ func BuildConfigKeyWithUser(keyPartial, userID string) string {
 }
 
 func isTokenExpired(bearerToken string) bool {
-	token, err := jwt.Parse(bearerToken, func(_jwtToken *jwt.Token) (interface{}, error) {
-		// uncomment when enabling local verification
-		// var kid *string
-		// if kidhdr, ok := _jwtToken.Header["kid"].(string); ok {
-		// 	kid = &kidhdr
-		// }
-
-		// publicKey, _, _, _ := util.ResolveJWTKeypair(kid)
-		// if publicKey == nil {
-		// 	msg := "failed to resolve a valid JWT verification key"
-		// 	if kid != nil {
-		// 		msg = fmt.Sprintf("%s; invalid kid specified in header: %s", msg, *kid)
-		// 	} else {
-		// 		msg = fmt.Sprintf("%s; no default verification key configured", msg)
-		// 	}
-		// 	return nil, fmt.Errorf(msg)
-		// }
-
-		return nil, nil
```

Comment:
Having the interface being returned here be `nil` causes the `jwt.Parse` method to always fail - if we want to parse the token without verifying we need to use `ParseUnverified`.